terraform/environments/electronic-monitoring-data/lambdas_iam.tf

```diff
@@ -437,24 +437,24 @@ data "aws_iam_policy_document" "output_fs_json_lambda_s3_policy_document" {
   }
 }
 
-resource "aws_iam_policy" "output_fs_json_lambda_s3_policy" {
-  name        = "output-fs-json-lambda-s3-policy"
-  description = "Policy for Lambda to use S3 for ${local.output_fs_json_lambda}"
-  policy      = data.aws_iam_policy_document.output_fs_json_lambda_s3_policy_document.json
+resource "aws_iam_policy" "extract_metadata_from_atrium_unstructured_s3_policy" {
+  name        = "extract-metadata-from-atrium-unstructured-lambda-s3-policy"
+  description = "Policy for Lambda to use S3 for extract_metadata_from_atrium_unstructured"
+  policy      = data.aws_iam_policy_document.extract_metadata_from_atrium_unstructured_s3_policy_document.json
 }
 
-resource "aws_iam_role_policy_attachment" "output_fs_json_lambda_s3_policy_attachment" {
-  role       = aws_iam_role.output_fs_json_lambda.name
-  policy_arn = aws_iam_policy.output_fs_json_lambda_s3_policy.arn
+resource "aws_iam_role_policy_attachment" "extract_metadata_from_atrium_unstructured_s3_policy_attachment" {
+  role       = aws_iam_role.extract_metadata_from_atrium_unstructured.name
+  policy_arn = aws_iam_policy.extract_metadata_from_atrium_unstructured_s3_policy.arn
 }
 
-resource "aws_iam_role_policy_attachment" "output_fs_json_lambda_vpc_access_execution" {
-  role       = aws_iam_role.output_fs_json_lambda.name
+resource "aws_iam_role_policy_attachment" "extract_metadata_from_atrium_unstructured_vpc_access_execution" {
+  role       = aws_iam_role.extract_metadata_from_atrium_unstructured.name
```

Comment:
thought to add this to our lambda module

Reply:
We can cover this in another, seperate PR if you want, not to muddy this already large PR